.travis.yml

```diff
@@ -6,14 +6,17 @@ language: c
 
 compiler:
   - gcc
-  - clang
 
 addons:
   apt:
     packages: [ python ]
 
-cache:
-  apt: true
+matrix:
+  include:
+    - os: linux
+      compiler: clang
```

Comment:
@chipitsine Frankly I don't really care about linux + clang since clang is already covered in OS X anyway.